Makefile

```diff
@@ -44,6 +44,7 @@ $(call add-crd-gen,ocsv1,./pkg/apis/ocs/v1,./deploy/crds,./deploy/crds)
 	gofmt \
 	golint \
 	govet \
```

Comment:
If these linters are included in the metalinter, they should be removed from the Makefile.